requirements.txt

```diff
@@ -1,3 +1,3 @@
 jinja2>=2.9,<3.0
-jsonschema>=2.6,<2.7
+jsonschema>=2.6,<3.4
```

Comment:
please increase the minimum version required to the 3.x range, the last release is 3.2.0 so the upper bound should be `3.3`: https://pypi.org/project/jsonschema/#history